tools/istio-iptables/pkg/dependencies/implementation.go

```diff
@@ -35,20 +73,80 @@ func (r *RealDependencies) execute(cmd string, redirectStdout bool, args ...stri
 	return externalCommand.Run()
 }
 
-// RunOrFail runs a command and panics, if it fails
+func (r *RealDependencies) executeXTables(cmd string, redirectStdout bool, args ...string) error {
```

Comment:
Why didn't you just wrap `execute`?

Reply:
There is a slight difference in dealing with stderr in the `execute` logic:
https://github.com/istio/istio/blob/020e7236ae230fece50ec079e05669a8a08666d1/tools/istio-iptables/pkg/dependencies/implementation.go#L71
https://github.com/istio/istio/blob/020e7236ae230fece50ec079e05669a8a08666d1/tools/istio-iptables/pkg/dependencies/implementation.go#L84
where we need to save `stderr` into a byte buffer for further usage instead of outputting it directly to stderr in `executeXTables`.

Does this make sense to you? Thanks!